src/SFML/Graphics/Text.cpp

```diff
@@ -31,15 +31,15 @@
 #include <SFML/Graphics/Color.hpp>
 #include <SFML/Graphics/ConvertTransform.hpp>
 #include <SFML/Internal.h>
+#include <SFML/Char32.hpp>
 
 
 ////////////////////////////////////////////////////////////
-sfText* sfText_create(void)
+sfText* sfText_create(const sfFont* font)
 {
-    sfText* text = new sfText;
-    text->Font = nullptr;
+    CSFML_CHECK_RETURN(font, nullptr);
 
-    return text;
+    return new sfText{ sf::Text{font->This}, font, {}, {}, {} };
```

Comment:
I like how this new pattern ensures all fields get initialized. I believe the previous code left certain things uninitialized.